inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/coroutine/ImageLoaderCoroutine.kt

```diff
@@ -0,0 +1,28 @@
+package com.rakuten.tech.mobile.inappmessaging.runtime.coroutine
+
+import android.graphics.Bitmap
+import com.squareup.picasso.Picasso
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+import timber.log.Timber
+import java.lang.Exception
+import kotlin.coroutines.CoroutineContext
+
+internal class ImageLoaderCoroutine(
+    private val coroutineContext: CoroutineContext = Dispatchers.IO
+) {
+
+    @SuppressWarnings("TooGenericExceptionCaught")
+    fun fetch(imageUrl: String): Bitmap? = runBlocking(coroutineContext) {
```

Comment:
Why use `runblocking` in coroutines?

Reply:
Ahh, sorry my bad, it was added to reach Picasso call from UT, updated the code to runBlocking only  in test.